cpp/util/etcd.h

```diff
@@ -2,6 +2,8 @@
 #define CERT_TRANS_UTIL_ETCD_H_
 
 #include <chrono>
+#include <glog/logging.h>
```

Comment:
SplitHosts was originally templated so you could have whichever container you wanted, and that was here for the `CHECK`s in the inlined code.